src/pages/home/sidebar/FloatingActionButtonAndPopover.tsx

```diff
@@ -454,6 +455,80 @@ function FloatingActionButtonAndPopover({onHideCreateMenu, onShowCreateMenu, isT
     const canModifyTask = canModifyTaskUtils(viewTourTaskReport, currentUserPersonalDetails.accountID);
     const canActionTask = canActionTaskUtils(viewTourTaskReport, currentUserPersonalDetails.accountID);
 
+    const menuItems = [
+        ...expenseMenuItems,
+        {
+            icon: Expensicons.ChatBubble,
+            text: translate('sidebarScreen.fabNewChat'),
+            onSelected: () => interceptAnonymousUser(startNewChat),
```

Comment:
Can you please retest the original issue that this interaction manager was added for is still fixed?